### PR TITLE
Fix latex rendering in tutorial

### PR DESCRIPTION
**Pull request by AtreyeeS**
Minor typo

### CODE REVIEW

File: examples/tutorials/analysis-1d/spectral_analysis_rad_max.py [hunk 1]
```diff
@@ -47,15 +47,15 @@
...
 should be used to define the size of the ON and OFF regions; \* in case
-an energy- (and offset-) dependent :math:`\theta` cut is applied, its
+an energy- (and offset-) dependent :math:`\\theta` cut is applied, its
 values are stored in additional `FITS` data unit, named
 ``RAD_MAX_2D` <https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/#rad-max-2d>`__.
 
 `Gammapy` provides a class to automatically read these values,
 `~gammapy.irf.RadMax2D`, for both cases (fixed or energy-dependent
-:math:`\theta` cut). In this notebook we will focus on how to perform a
+:math:`:math:`\\theta` cut). In this notebook we will focus on how to perform a
```
Comment (QRemy) [inline examples/tutorials/analysis-1d/spectral_analysis_rad_max.py:56]: This line does not match the other changes, `:math: seems repeated

```suggestion
:math:`\\theta` cut). In this notebook we will focus on how to perform a
```

**Review by QRemy — CHANGES_REQUESTED** [review]
Thanks @AtreyeeS, I added a suggestion to fix a typo.

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS !

**Comment by registerrier** [comment]
CI fail is unrelated. Merging.

**Pull request merged**